jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -585,3 +585,97 @@ def _fft_rng_factory(dtype):
           [arg, StaticArg(np.array([shift_amount], dtype=dtype))])
   for arg, dtype, shift_amount in shift_inputs
 )
+
+lax_select_and_gather_add = tuple(
```

Comment:
I can't tell why we have these different groups of harnesses. I think that a short comment for each might be useful.

Reply:
Added comment pointing to `tests.lax_autodiff_test.testReduceWindowGrad` – from which the tests are taken in `cb32206ee3`.